src/main/java/io/gravitee/reporter/elasticsearch/BeanRegister.java

```diff
@@ -49,7 +50,7 @@ public boolean registerBeans(ElasticsearchInfo info, ReporterConfiguration confi
 
     private AbstractElasticBeanRegistrer getBeanRegistrerFromElasticsearchInfo(ElasticsearchInfo elasticsearchInfo) {
         if (elasticsearchInfo.getVersion().isOpenSearch()) {
-            if (elasticsearchInfo.getVersion().getMajorVersion() == 1) {
+            if (Set.of(1, 2).contains(elasticsearchInfo.getVersion().getMajorVersion())) {
```

Comment:
For best visibility, it would be better to have a proper constant such as `SUPPORTED_OPENSEARCH_MAJOR_VERSION` or something. WDYT?